tsup.config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "entry": ["src/index.ts", "src/utils.ts"],
+  "splitting": false,
+  "sourcemap": true,
+  "dts-resolve": true,
+  "clean": true,
+  "format": ["cjs", "esm"],
+  "treeshake": true,
```

Comment:
Nice to have added tree shaking! 👌

Reply:
Oh we don't actually have it right now, because I reverted tsup. I will add it in a new PR but I need to figure out how to keep the build/main and build/module paths. By default tsup uses a common path for both and usually dist. That's where that came from, but I agree that would be a good thing.